client/index.html

```diff
@@ -115,14 +115,16 @@ <h4 id="status"><b>Status: {{status}}</b></h4>
 									{{/if}}
 								{{/if}}
 							{{else if user.applied}}
-								<p>You've submitted your application and are all set for now!</p>
+								<p>You've submitted your application! We are currently reviewing your application and will get back to you in late August.</p>
 								<p>Application type: <b>{{user.applicationBranch}}</b></p>
 								{{#ifCond user.applicationBranch "Participant"}}
-									<p>For HackGT: Into The Rabbit Hole, participant applications will be considered in two rounds.</p>
-									<ul>
-										<li><strong>Travel Reimbursement Deadline</strong>: Wendesday, August 14th 2019 at 11:59 pm EDT</li>
-										<li><strong>Non-Travel Deadline</strong>: Friday, September 20th 2019 at 11:59 pm EDT</li>
-									</ul>
+									{{#if applicationStatus.areOpen}}
+										<p>For HackGT: Into The Rabbit Hole, participant applications will be considered in two rounds.</p>
+										<ul>
+											<li><strong>Travel Reimbursement Deadline (Closed)</strong>: <strike>Wendesday, August 14th 2019 at 11:59 pm EDT</strike></li>
```

Comment:
See suggestions on MDN for deprecated `<strike>` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strike